Library/Formula/kettle.rb

```diff
@@ -37,6 +37,8 @@ def plist; <<-EOS.undent
           <key>KETTLE_HOME</key>
           <string>#{etc}/kettle</string>
         </dict>
+        <key>WorkingDirectory</key>
+        <string>/usr/local/etc/kettle</string>
```

Comment:
`#{etc}/kettle` as you can't assume Homebrew is installed in `/usr/local`, sadly.